packages/client-browser/tsconfig.client-browser.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "outDir": "dist2"
```

Comment:
What's for?

Reply:
Rudimentary leftover from my experiments with workspaces